### PR TITLE
[DOCS] add text about eth.syncing status check in mining tutorial

### PR DESCRIPTION
**Pull request by thadguidry**
-  this helps to align and address the 90% question often asked around the internet and our Discord and why we wrote more helpful detail in our FAQ about syncing.

### CODE REVIEW

**Comment by xela274** [comment]
Y

File: docs/_interface/mining.md [hunk 1]
```diff
@@ -47,6 +47,29 @@ Now start geth and wait for it to sync the blockchain. This will take quite a wh
...
+
+    {
+      currentBlock: 10707814,
+      highestBlock: 13252182,
+      knownStates: 0,
+      pulledStates: 0,
+      startingBlock: 3809258
+}
+
+A response from `eth.syncing` of `false` means you are synced up.
```
Comment (verheesj) [inline docs/_interface/mining.md:71]: a `false` could also mean that you are not syncing and not synced up?

**Review by verheesj — COMMENTED** [review]
eth.syncing is not an indicator of being fully synced?

File: docs/_interface/mining.md (hunk 1 above)
Comment (orothad) [reply docs/_interface/mining.md:71]: ah, very good point.  You can fix that perhaps by expanding that sentence I added.
But that sentence is not the API reference itself which covers that detail and doesn't say much:  https://eth.wiki/json-rpc/API#eth_syncing  <-- could probably be improved by someone

**Pull request merged**